src/Utility.js

```diff
@@ -55,10 +55,11 @@ export function replace (value, pattern, replacement) {
 /**
  * @param {string} value
  * @param {string} search
+ * @param {number} position
  * @return {number}
  */
-export function indexof (value, search) {
-	return value.indexOf(search)
+export function indexof (value, search, position) {
```

Comment:
perhaps now this function stops being homomorphic, should I pass `0` as an extra argument at all other call sites? 🤔

Reply:
Yes